.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,206 @@
+Hi, thank you for your interest in contributing to the WordPress Theme Review Coding Standards! We look forward to working with you.
+
+# Reporting Bugs
+
+Before reporting a bug, you should check what sniff an error is coming from.
+Running `phpcs` with the `-s` flag will show the name of the sniff with each error.
+
+Bug reports containing a minimal code sample which can be used to reproduce the issue are highly appreciated as those are most easily actionable.
+
+# Contributing patches and new features
+
+## Branches
+
+Ongoing development will be done in the `develop` branch with merges done into `master` once considered stable.
+
+To contribute an improvement to this project, fork the repo and open a pull request to the `develop` branch. Alternatively, if you have push access to this repo, create a feature branch prefixed by `feature/` and then open an intra-repo PR from that branch to `develop`.
+
+Once a commit is made to `develop`, a PR should be opened from `develop` into `master` and named "Next release". This PR will provide collaborators with a forum to discuss the upcoming stable release.
+
+# Considerations when writing sniffs
+
+## Public properties
+
+When writing sniffs, always remember that any `public` sniff property can be overruled via a custom ruleset by the end-user.
+Only make a property `public` if that is the intended behaviour.
+
+When you introduce new `public` sniff properties, or your sniff extends a class from which you inherit a `public` property, please don't forget to update the [public properties wiki page](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/wiki/Customizable-sniff-properties) with the relevant details once your PR has been merged into the `develop` branch.
+
+# Unit Testing
+
```

Comment:
"_All PRs which affect sniffs, whether bug fixes to existing sniffs, or the addition of a new sniff, should be accompanied by unit tests._"

~~Maybe we should even have a little section about sniff names, categories, unit test case files and unit test files in general.~~ Found it! (see below)

Aside from a section on how to categorize a new sniff in the first place, seeing how difficult that turned out to be in the original categorization.